FrejaEidClient/src/test/java/com/verisec/frejaeid/client/client/impl/SignClientGetResultTest.java

```diff
@@ -128,11 +134,41 @@ public void getSignResultOrganisational_expectSuccess() throws FrejaEidClientInt
                 .send(FrejaEnvironment.TEST.getUrl() + MethodUrl.ORGANISATION_SIGN_GET_ONE_RESULT,
                       RequestTemplate.SIGN_RESULT_TEMPLATE, signResultRequest, SignResult.class, RELYING_PARTY_ID);
         Assert.assertEquals(SIGN_REFERENCE, response.getSignRef());
-        Assert.assertEquals(TransactionStatus.STARTED, response.getStatus());
+        Assert.assertEquals(TransactionStatus.APPROVED, response.getStatus());
         Assert.assertEquals(SIGN_DETAILS, response.getDetails());
         Assert.assertEquals(REQUESTED_ATTRIBUTES, response.getRequestedAttributes());
     }
 
+    @Test
+    public void getSignResultOrganisational_withAdditionalAttributes_expectSuccess() throws FrejaEidClientInternalException, FrejaEidException {
```

Comment:
MEDIUM
Add test for initiating add org id transaction with additional attributes and display type.

Reply:
Done